### ISSUE TITLE
Urinary mulberry cells

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4880]
**Preferred term label:**
Urinary mulberry cells

**Synonyms**

**Definition (free text, please give PubMed ID)**
Distal tubular epithelial cells in which globotriaosylceramide (Gb3) has accumulated. they are the characteristic feature of Fabry disease. Urinary mulberry bodies are a component of mulberry cells that can be distinguished easily from fat particles by their inner lamellar appearance. 

Comment
See Figure 1 of PMID:28593486.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0012614

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fabry disease

**Your nano-attribution (ORCID)**
